src/game.zig

```diff
@@ -362,6 +363,61 @@ pub const collision = struct {
 		return resultingMovement;
 	}
 
+	fn isBlockIntersecting(block: Block, posX: i32, posY: i32, posZ: i32, center: Vec3d, extent: Vec3d) bool {
+		const model = &models.models.items[block.mode().model(block)];
+		const position = Vec3d{@floatFromInt(posX), @floatFromInt(posY), @floatFromInt(posZ)};
```

Comment:
Not a requirement: You could also do `const position: Vec3d = @floatFromInt(Vec3i{posX, posY, posZ});`